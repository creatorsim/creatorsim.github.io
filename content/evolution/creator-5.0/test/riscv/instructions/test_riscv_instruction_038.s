
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    li x5, 16
    li x6, 256
    li x7, -8192
    li x8, 24

    div x9, x7, x5
    div x10, x6, x5
    div x11, x7, x8
    div x12, x8, x7