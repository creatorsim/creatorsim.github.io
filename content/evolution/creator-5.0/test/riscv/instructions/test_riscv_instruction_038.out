
cr[PC]:0x24; ir[x5,t0]:0x10; ir[x6,t1]:0x100; ir[x7,t2]:0xffffe000; ir[x8,fp,s0]:0x18; ir[x9,s1]:0xfffffe00; ir[x10,a0]:0x10; ir[x11,a1]:0xfffffeab; keyboard[0x0]:''; display[0x0]:''; 

